README.md

```diff
@@ -70,10 +70,10 @@ Then you can run the commands below:
 yarn
 
 # serve with hot reload at localhost:9080
```

Comment:
I even don't know what this sentence means.
But we need to check if it is still relevant.

Reply:
if you cloned the repo the node_modules folder is not there so you need to run yarn in order do install all the packages that mentioned at the package.json

Reply:
Sorry, I didn't talk about `yarn`.
I'm talking about `# serve with hot reload at localhost:9080`. But after some documentation reading, I guess it is still relevant.